src/sentry/integrations/msteams/webhook.py

```diff
@@ -200,6 +202,55 @@ def handle_member_removed(self, request):
         integration.delete()
         return self.respond(status=204)
 
+    def make_action_data(self, identity, data):
+        action_data = {}
+        if data["actionType"] == "unresolve" or data["actionType"] == "unignore":
+            action_data = {"status": "unresolved"}
+            analytics.record("integrations.msteams.unresolve", actor_id=identity.user_id)
+        elif data["actionType"] == "resolve":
+            status = data["resolveInput"]
+            status_data = status.split(":", 1)
+            resolve_type = status_data[-1]
+
+            action_data = {"status": "resolved"}
+            if resolve_type == "inNextRelease":
+                action_data.update({"statusDetails": {"inNextRelease": True}})
```

Comment:
Personally not a fan of `update` cause then you need to have context on what `action_data` used to be. I'd suggest:
```
action_data = {
    "status": ...,
    "statusDetails": ...
}
```

Reply:
Is the definition of `action_data` not close enough to the `update` statement to give the context you're talking about?

Reply:
It is close and shouldn't be hard to read for this function. Just talking more generally. Upto you though. This is a nit :).